app/javascript/components/server-components/Admin/Nav.tsx

```diff
@@ -62,8 +62,13 @@ export const Nav = ({ title, current_user }: Props) => {
           icon="envelope-open-fill"
           href={Routes.admin_unblock_email_domains_url(routeParams)}
         />
-        <NavLink text="Sidekiq" icon="lighting-fill" href={Routes.admin_sidekiq_web_url(routeParams)} />
-        <NavLink text="Features" icon="solid-flag" href={Routes.admin_flipper_ui_url(routeParams)} />
+        <NavLink
+          text="Sidekiq"
+          icon="lighting-fill"
+          href={Routes.admin_sidekiq_web_url(routeParams)}
+          dataTurbo={false}
```

Comment:
Why are these false?

Reply:
Because they point to external UIs: the Sidekiq and the Flipper dashboard, that have their own design layout.

Reply:
Should data-turbo be false by default and it should be set to true? It would remove the need for these, and may be more intuitive behavior.